packages/collector/test/tracing/cloud/gcp/storage/test.js

```diff
@@ -49,11 +50,14 @@ if (
   });
 } else {
   let mochaSuiteFn;
+  // There is bug in node 21.2.0, more details please see
+  // https://github.com/googleapis/nodejs-storage/issues/2368
 
-  if (!supportedVersion(process.versions.node) || !process.env.GCP_PROJECT) {
+  if (!supportedVersion(process.versions.node) || !process.env.GCP_PROJECT ||
+  semver.gte(process.versions.node, '21.2.0')) {
```

Comment:
@instana/eng-nodejs I have created a ticket for this. The bug is resolved. Good first task